knexfile.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/cadence_dev'
```

Comment:
I've never used trailing commas before... I thought that caused a syntax error? But I guess not.